docs/source/cli.rst

```diff
@@ -4,9 +4,7 @@ Command-Line Interface
 ======================
 
 The MLflow command-line interface (CLI) provides a simple interface to various functionality in MLflow. You can use the CLI to run projects, start the tracking UI, create and list experiments, download run artifacts,
-serve MLflow Python Function and scikit-learn models, and serve models on
-`Microsoft Azure Machine Learning <https://azure.microsoft.com/en-us/services/machine-learning-service/>`_
```

Comment:
You can still use the `mlflow deployments` CLI to deploy on AzureML after installing the `azureml-mlflow` plugin, so we should leave this here.

Reply:
reverted!